src/runtime/plugins/register.ts

```
@@ -318,23 +320,46 @@ const vRegisterSelect: RegisterSelectCustomDirective = {
 }
 
+function getBaseValue(value: RegisterValue, el: HTMLSelectElement) {
+  const externalValue = value.innerRef.value
+  const options = el.options
+  if (typeof externalValue === 'string') {
+    const selectedOption = options[options.selectedIndex]
+    return selectedOption?.value ?? ''
+  }
+
+  const optionsArray = [...options]
+  if (externalValue instanceof Array) {
```

Biome (1.9.4)

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)